src/test/kotlin/com/buildit/bookit/v1/location/bookable/BookableControllerMockMvcTests.kt

```diff
@@ -54,19 +59,32 @@ class BookableControllerMockMvcTests @Autowired constructor(
             .build()
     }
 
-    @BeforeEach
-    fun setupMocks() {
-        whenever(locationRepo.getLocations())
-            .doReturn(listOf(Location("location-guid", "NYC", "America/New_York")))
-        whenever(bookableRepo.getAllBookables())
-            .doReturn(listOf(Bookable("bookable-guid", "guid", "The best bookable ever", Disposition())))
-    }
-
     @Test
     fun getExistingBookableTest() {
-        mvc.perform(MockMvcRequestBuilders.get("/v1/location/location-guid/bookable/bookable-guid"))
+        mvc.perform(MockMvcRequestBuilders.get("/v1/location/${location.id}/bookable/${bookable.id}"))
             .andExpect(status().isOk)
-            .andExpect(jsonPath<String>("$.name", equalToIgnoringCase("The best bookable ever")))
+            .andExpect(jsonPath<String>("$.name", equalToIgnoringCase(bookable.name)))
             .andExpect(jsonPath<Boolean>("$.disposition.closed", equalTo(false)))
     }
+
+    @TestConfiguration
+    class InternalConfig {
```

Comment:
Any possibility of DRY?

Reply:
you mean between tests?  although they'd have to share test data assumptions...

Reply:
Location's mapping is defined in multiple tests.  Just wonder if the @TestConfiguration can be DRY for that Entity, for example.  It's not a major thing -- the code is write once (in a few locations), and never revisit.

Reply:
It might be a stupid question, too.  I'm really not familiar with Spring Data beyond the benefits, WRT repos, etc.  The magical parts and implementation niceties are still a mystery.

Reply:
i think it's for the test...although there mgiht be some ways to factor stuff out